controllers/capabilities/redis_threescale_reconciler.go

```diff
@@ -0,0 +1,80 @@
+package controllers
+
+import (
+	"context"
+	"fmt"
+	"github.com/3scale/3scale-operator/pkg/common"
+	"github.com/3scale/3scale-operator/pkg/reconcilers"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"reflect"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+)
+
+type RedisConfigCMReconciler struct {
+	*reconcilers.BaseReconciler
+	previousConfigMap *corev1.ConfigMap
```

Comment:
usually this approach is not recommended. The state of a controller needs to be read from the cluster... cannot rely on local status. The rationale being what if the operator controller is being restarted?

Reply:
but I will continue with your suggestion wihout new controller. Thank you

Reply:
done. Thank you